justfile

```diff
@@ -4,9 +4,6 @@ set dotenv-load := false
 # @ - Quiet recipes (https://github.com/casey/just#quiet-recipes)
 # _ - Private recipes (https://github.com/casey/just#private-recipes)
 
-IS_PROD := env_var_or_default("PROD", env_var_or_default("IS_PROD", ""))
-# `PROD_ENV` can be "ingestion_server" or "catalog"
-PROD_ENV := env_var_or_default("PROD_ENV", "")
```

Comment:
These variables were used long ago when we used to deploy the ingestion server and catalog using `just`. We don't do that any more, and they just add noise/complexity to this file now, so are good to remove.